### ISSUE TITLE
Add CSV download

### ISSUE THREAD

**Issue by njvack** [njvack/scatterize#32]
"What?" you ask yourself, "but I put a CSV in!"

Yup. What we want out is the model results. Partly this is because you should be able to see this; partly it's for accessibility reasons -- this is a non-graphical way to get the data. Columns should be:

X (titled with axis label), Y (similarly), [orig_Y] (if residualized), [orig_X, orig_Y] (for Spearman), [nuisance1, ... nuisanceN, residual] (for parametric stats), color (if color is active), [smooth_low, smooth_line, smooth_high] (if smoothing is active)

This should go with our downloads. Those are currently in the "Share" button's dialog but that should probably be re-labeled "Download"
